### PR TITLE
Carambola Adapter

### PR DESCRIPTION
**Pull request by LiranMotola**
## Type of change

- [x] New bidder adapter 

## Description of change
New adapter submitted

- test parameters for validating bids
```
{
    bidder: 'carambola',
    adId: "2e3daacdeed03d",
    requestId: 'c9ad932a-41d9-4821-b6dc-0c8146029faf',
    adUnitCode:"cbola_prebid_code_97",
    token:"CGYCLyIy",
    pageViewId:"22478638",
    params: {
      pid: 'hbtest',
      did: 110000,
      wid: 0
    }
  }
```
- contact email of the adapter’s maintainer
- [X] Liran@carambola.com

### CODE REVIEW

**Comment by protonate** [comment]
Thanks for the contribution, I'm unsure about the test params, these appear to be "bid-specific" rather than configuration for your adapter:
```
    adId: "2e3daacdeed03d",
    requestId: 'c9ad932a-41d9-4821-b6dc-0c8146029faf',
    adUnitCode:"cbola_prebid_code_97",
    token:"CGYCLyIy",
    pageViewId:"22478638",
```

File: src/adapters/carambola.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+        }
+      }
+
+      let server = bid.params.server || 'route.carambo.la';
+      let cbolaHbApiUrl = '//' + server + '/' + REQUEST_PATH;
+
+      //  the responses of the bid requests
+      ajax(cbolaHbApiUrl + _jsonToQueryString(tempParams), response => {
+        //  no response
+        if (!response || response.cpm <= 0) {
```
Comment (protonate) [inline src/adapters/carambola.js]: `response` here is still a string and needs to be `JSON.parse`d before you can access properties e.g. `response.cpm`

Comment (LiranMotola) [inline src/adapters/carambola.js]: Thank you. indeed there was an error. fixed now.

**Comment by protonate** [comment]
Hi @LiranMotola I'm still not getting bids back and have questions about the need for the bidder settings config that is "bid specific".

**Comment by LiranMotola** [comment]
Hi @protonate , the following params will get you a valid bid response now:
 params: {pid: 'hbtest',  did: 112591, wid: 0 }
Could you please elaborate on the questions you have about the "bid specific" ?
Thanks

**Comment by protonate** [comment]
Thanks, confirmed bids back, reviewed.

**Pull request merged**